plugins/PostProcessingPlugin/PostProcessingPlugin.py

```diff
@@ -173,7 +173,7 @@ def _createView(self):
         Logger.log("d", "Creating post processing plugin view.")
 
         ## Load all scripts in the scripts folders
-        for root in [PluginRegistry.getInstance().getPluginPath("PostProcessingPlugin"), Resources.getStoragePath(Resources.Preferences)]:
+        for root in [PluginRegistry.getInstance().getPluginPath("PostProcessingPlugin"), Resources.getStoragePath(Resources.Resources)]:
```

Comment:
That can be fixed with a settings upgrade function (which could move the scripts to the correct folder).

Reply:
It's not so easy, since the post-processing scripts don't have a version number in them, which is required by the version upgrade system. That is to say, they have a version number in their settings JSON, but that version number is tied to the version number in the printer definition files (.def.json) due to the way those settings are loaded.

I think the best/easiest solution is to load from both directories but document only the directory in the `.local` folder.